app/globals.css

```diff
@@ -31,8 +31,9 @@
     --destructive: 0 84.2% 60.2%;
     --destructive-foreground: 0 0% 98%;
 
-    --corpo-background: 201 70% 31%;
-    --corpo-header-background: 360 55% 48%;
+    --corpo-background: 199.1 38.8% 42.9%;
+    --corpo-header-background: 205.1 54.5% 80.2%;
+    --corpo-chatpanel-background: 205.1 54.5% 80.2%;
```

Comment:
"chatpanel" is technically the space behind where the agent/human chat messages appear.